modules/swagger-codegen/src/main/resources/php/Configuration.mustache

```diff
@@ -123,51 +109,6 @@ class Configuration
     protected $tempFolderPath;
 
     /**
-     * Indicates if SSL verification should be enabled or disabled.
```

Comment:
> We might make 2) the default to keep BC though.

That's what I prefer - a reasonable default so that new users can simply generate a functional PHP client with the default setting (and they can later customize the output based on their requirements)